tutorials/applications/Simulating Sequences with Errors and Noise.ipynb

```diff
@@ -0,0 +1,1129 @@
+{
```

Comment:
<p>Maybe mention you set <code>eta=0</code>  so that you only have measurement errors.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pasqal-io/Pulser/pull/232/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>